README.md

```diff
@@ -11,32 +11,32 @@ Track your daily step count, stay healthy and fight the climate change, one step
   - [🦁 Table of Contents](#-table-of-contents)
   - [🌳 Inspiration](#-inspiration)
   - [🥕 Features](#-features)
-  - [🐻‍❄️ Instalation and First Launch](#️-instalation-and-first-launch)
+  - [🐻‍❄️ Installation and First Launch](#️-instalation-and-first-launch)
   - [🪴 Technologies](#-technologies)
 
 ## 🌳 Inspiration
 
-A couple years ago together with my friends I took a part in a programming competition. The objective was to build a mobile app that solves a global problem. We didn't win, but the app we built quickly spreaded in our families.
+A couple of years ago together with my friends, I took a part in a programming competition. The objective was to build a mobile app that solves a global problem. We didn't win, but the app we built quickly spread in our families.
 
 ## 🥕 Features
 
 <img src="https://user-images.githubusercontent.com/60577942/221682705-39a0e476-bb52-4257-8d3b-5e5a64e72424.gif" alt="application demo" width="270">
 
 <br />
 
-- Track the step count, burned calories, distance travelled and CO₂ saved
-- Get rewarded by completing your daily goal and keep motivated
-- Use a handful notification with your daily stats updated live
+- Track your step count, burnt calories, distance travelled and CO₂ saved
+- Get rewarded by completing your daily goal and stay motivated
+- View handy notifications with, daily stats updated live
```

Comment:
Is this comma intentional?

Reply:
oh my bad sorry it looks weird now re reading it :(